rust/Cargo.toml.in

```diff
@@ -65,6 +65,10 @@ ldap-parser = { version = "~0.4.0" }
 
 time = "~0.3.36"
 
+# pinned until oss-fuzz supports newer rust nightly cf https://github.com/google/oss-fuzz/pull/12365
+serde = { version = "=1.0.197" }
```

Comment:
Is this `=` as a part of version string intentional? What does it mean?

Reply:
yes, pin exact version https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#comparison-requirements